<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>run_view</class>
 <widget class="QDialog" name="run_view">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>666</width>
    <height>93</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Run</string>
  </property>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>671</width>
     <height>91</height>
    </rect>
   </property>
   <widget class="QPushButton" name="show_query_progress_button">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>151</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>Show query progress</string>
    </property>
   </widget>
   <widget class="QPushButton" name="show_stage_progress_button">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>151</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>Show stage progress</string>
    </property>
   </widget>
   <widget class="QPushButton" name="show_retrieve_progress_button">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>161</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>Show retrieve progress</string>
    </property>
   </widget>
   <widget class="QPushButton" name="show_process_progress_button">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>20</y>
      <width>161</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>Show process progress</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>3</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
